File: skills/sn-init/templates/lang/py/Makefile.py
# Appended to the project Makefile by sn-init for --lang=py.
.PHONY: install fmt lint test mcp-server agent-run

install:
	uv sync --all-extras

fmt:
	uv run ruff format src tests

lint:
	uv run ruff check src tests

test:
	uv run pytest

agent-run:
	uv run python src/agent.py

mcp-server:
	uv run python mcp_server/main.py

File: skills/sn-init/templates/lang/go/Makefile.go
# Appended to the project Makefile by sn-init for --lang=go.
.PHONY: install fmt lint test mcp-server agent-run

install:
	go mod tidy

fmt:
	go fmt ./...

lint:
	go vet ./...

test:
	go test ./...

agent-run:
	go run ./src

mcp-server:
	go run ./mcp_server
